src/test/codegen/asm-target-clobbers.rs

```diff
@@ -0,0 +1,21 @@
+// only-x86_64
```

Comment:
`--target=x86_64-...` and `needs-llvm-components: x86` would ensure this test also runs on developer machines that aren't x86_64.

Reply:
That wouldn't actually work since the x86_64 libstd isn't built.

Reply:
Ah, I was meaning to implicitly suggest `no_core`, but I forgot the `asm!` macro is a `libcore` thing in the first place... nvmd then.